index.md

```diff
@@ -178,36 +178,33 @@ FIXME
 
 ### Goals
 
-1.  Again, the easier it is for people to collaborate, the more likely they are to do so
-    (and to give you credit)
-2.  Two most reported barriers from [Steinmacher et al][steinmacher-newcomers]
-    are finding a task to start on
-    and problems setting up the local workspace to start work
-    *   So remove those
-3.  Additional disincentive is uncertainty: what am I allowed to do?
-    *   So remove that as well
-    *   Lack of an explicit license implies the author is keeping all rights
-        and others are not allowed to re-use
+You may work on your project with others, your known collaborators. But, even at an early stage, you may be interested to open your project so that you can attract new, as yet unknown collaborators. This section list approaches for making it straightforward for others to start collaborating on your project.
+
+1.  *Simplicity:* the easier it is for people to collaborate, the more likely they are to do so (and to give you credit)
+2.  *Low entry* Remove the two most reported barriers to contributing from [Steinmacher et al][steinmacher-newcomers]:
+    * finding a task to start on
+    * setting up the local workspace to start work
+3.  *Clarity:* remove uncertainty around what a potential collaborator is allowed to do
 
 ### Rules
 
-1.  Every project has a short README file explaining its purpose
-    *   Includes a contact address that actually works
-2.  And a plain text file (often called notes.txt or todo.txt) containing the to-do list
-    *   Aimed at contributors, where README is aimed at users
-3.  And a LICENSE file
+1.  Have a short README file explaining the project's purpose.  
+This file should includes contact information that actually works. This file is often the first thing users of your project will look at, so make it explicit already here that you welcome contributors and point them to the ways to help out. Consider using a CONTRIBUTION file to describe any steps needed to start working: dependencies that need to be installed, running of (unit) tests, guidelines or rules that your project adheres to (e.g. on commit messages or checklists you may use before accepting a suggested change).
+2.  Have a plain text file containing the to-do list.  
```

Comment:
Rephrase this as "have a shared public to-do list", and then talk about using a text file vs. GitHub issues as implementations?